README.md

```
@@ -1 +1,167 @@
+- ## Kafka 도입
+  
+- ## Redis 도입
+
+<aside>
+
+
+# 🔬 트러블슈팅
+
+</aside>
+
+- ## Running-Record 생성 API 처리량 개선
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)